test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -649,3 +633,185 @@
       taintNodes: 1000
       normalNodes: 4000
       measurePods: 4000
+
+# These test cases use ResourceClaims that get created in advance,
+# with deterministic naming. The deterministic names are required
+# for the references between objects.
+#
+# They are currently commented out because of the additional time
```

Comment:
maybe we can add some kind of flag "experimental" or "wip" that prevents them from running by default?

Reply:
Comment removed. I also renamed the test cases a bit: "InlineResourceClaim" wasn't quite accurate.

> Should we consider moving these tests into the dra folder?

I prefer to keep them in the main `performance-config.yaml` because then they'll automatically be included in the normal Prow jobs.